packages/compass-aggregations/src/components/focus-mode/focus-mode-stage-editor.tsx

```diff
@@ -20,6 +19,13 @@ const headerStyles = css({
   justifyContent: 'flex-start',
   gap: spacing[2],
   marginLeft: spacing[2],
+  marginRight: spacing[2],
+  flexWrap: 'wrap',
+});
+
+const linkStyles = css({
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
```

Comment:
Any particular reason we are adding overflow hidden here? If you want to keep it, you need to add some padding on top (and bottom so that center alignment still works) too to account for the icon, otherwise it is cut when this link wraps:

<img width="196" alt="image" src="https://user-images.githubusercontent.com/5036933/216041900-81dd8810-719b-4ab6-ae2f-976005033948.png">